### PR TITLE
[WIP] runtime-rs: support nydus v6 as rootfs of container

### PR DESCRIPTION
**Pull request by Leospard**
Depend on kernel patch where nydus v6 could also be handled by the guest kernel.
With that patch, bootstrap and blobs are shared with guest VM via passthroughfs and blobfs so that they can be mounted inside guest.

### CODE REVIEW

**Comment by imeoer** [comment]
@Leospard Can we give some usage docs?

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ src/agent/protocols/src/*.rs
 !src/agent/protocols/src/lib.rs
 build
 src/tools/log-parser/kata-log-parser
+.DS_Store
```
Comment (imeoer) [inline .gitignore:18]: Redundant file.

File: src/runtime-rs/crates/resource/src/rootfs/nydus_rootfs.rs [hunk 2]
```diff
@@ -124,6 +130,72 @@ impl NydusRootfs {
                     rootfs_guest_path,
                 ))
             }
+            NYDUS_ROOTFS_V6 => {
+                // enable dax flag for bootstrap and blobfiles.
+                let dax_threshold_size_kb: u64 = 0;
```
Comment (Apokleos) [inline src/runtime-rs/crates/resource/src/rootfs/nydus_rootfs.rs:135]: Why `dax_threshold_size_kb` with hardcode `0` ?

One more question but important: does it need to check DAX whether it's enabled ?

**Review by Apokleos — COMMENTED** [review]
Thx @Leospard some comments!

**Pull request closed without merging**